### PR TITLE
Bump llvm-project@45964eb9b88c46045e4e84beb4e2135cdeed6855

### PR DESCRIPTION
**Pull request by rsuderman**
Includes bump of torch-mlir@8995c90879568ba9c04e13bc7faa70be035d3d7b

### CODE REVIEW

File: build_tools/pkgci/external_test_suite/onnx_cpu_llvm_sync.json
Comment (ScottTodd) [inline build_tools/pkgci/external_test_suite/onnx_cpu_llvm_sync.json:1]: Oh, I may have broken the config file uploading logic: https://github.com/iree-org/iree/actions/runs/9355995890/job/25752745889?pr=17554#step:10:22
`Warning: No files were found with the provided path: ${CONFIG_FILE_PATH}. No artifacts will be uploaded.`

you should be able to at least copy/paste from the previous step though: https://github.com/iree-org/iree/actions/runs/9355995890/job/25752745889?pr=17554#step:9:22

Comment (ScottTodd) [reply build_tools/pkgci/external_test_suite/onnx_cpu_llvm_sync.json:1]: Testing a fix for that. When it is working you should just be able to download the new .json files from the job summary page. For now you can copy the json from the logs (no need to read through the output and update test statuses manually)

Comment (ScottTodd) [reply build_tools/pkgci/external_test_suite/onnx_cpu_llvm_sync.json:1]: https://github.com/iree-org/iree/pull/17557

**Review by ScottTodd — APPROVED** [review]
LGTM assuming no major benchmark regressions

**Comment by ScottTodd** [comment]
Benchmarks were running already :P ah well
https://github.com/iree-org/iree/actions/runs/9357220505/attempts/1

**Comment by MaheshRavishankar** [comment]
> Benchmarks were running already :P ah well https://github.com/iree-org/iree/actions/runs/9357220505/attempts/1

Oh, I didnt see them in the list of checks. Thats why I added the label. Ok, next time, ill wait.

**Comment by ScottTodd** [comment]
Looks like all checks are passing now.

**Pull request merged**